packages/dashboard/src/components/rmf-app/rmf-app.tsx

```diff
@@ -230,6 +236,41 @@ function TaskContextsProvider(props: React.PropsWithChildren<{}>): JSX.Element {
   return <TasksContext.Provider value={tasks}>{props.children}</TasksContext.Provider>;
 }
 
+function RmfHealthContextsProvider(props: React.PropsWithChildren<{}>): JSX.Element {
+  const doorStates = React.useContext(DoorStateContext);
+  const liftStates = React.useContext(LiftStateContext);
+  const dispenserStates = React.useContext(DispenserStateContext);
+  const robotStates = React.useContext(FleetStateContext);
```

Comment:
This is very detrimental to performance, if 10 items send at 10hz, we are already doing 10,000 calculations per second.